integrations/fathom/src/fathomScript.raw.js

```diff
@@ -22,10 +22,13 @@
         }
         return '';
     }
+    const disableCookies = getCookie(GRANTED_COOKIE) !== 'yes';
 
     // Function to track external links
     function trackExternalLink(event) {
-        const disableCookies = getCookie(GRANTED_COOKIE) !== 'yes';
+        if (disableCookies) {
+            return;
+        }
```

Comment:
@addisonschultz Same here, I suggested the wrong change.

This should be
```js
function trackExternalLink(event) {
      if (getCookie(GRANTED_COOKIE) !== 'yes') {
          return;
      }
...
```

And you can get rid of 
```js
    const disableCookies = getCookie(GRANTED_COOKIE) !== 'yes';
```